bin/deploy

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+"""
+Deploy a dotcom build to the production Elastic Beanstalk servers.
+
+Usage:
+    deploy <sha>
+    deploy -h | --help
+
+Arguments:
+    <sha>  The hash of a commit that has been deployed to dotcom-dev
+"""
+
+from docopt import docopt
+import boto3
+
+
+def main(args):
+  environment_name = "dotcom-prod"
```

Comment:
Consider making this a positional argument; I think it's worth requiring the user to explicitly specify the environment that they want to deploy to, so as to make the action more deliberate. Also then it can be used for dev-green as well.

Reply:
👍 that makes sense.